Code/Editor/Lib/Tests/Camera/test_EditorCamera.cpp

```diff
@@ -584,6 +707,14 @@ namespace UnitTest
             const bool actualCameraGoToPositionInstantly = SandboxEditor::CameraGoToPositionInstantlyEnabled();
             EXPECT_THAT(expectedCameraGoToPositionInstantly, Eq(actualCameraGoToPositionInstantly));
         }
+
+        {
+            const float existingCameraGoToPositionDuration = SandboxEditor::CameraGoToPositionDuration();
+            const float expectedCameraGoToPositionDuration = !existingCameraGoToPositionDuration;
```

Comment:
Using ! with a float reads a bit weird to me.

Reply:
It should be ` + 1.0f` like the others (just to show it's changed) - so crazy this compile without issue 😅